primap2/csg/__init__.py

```diff
@@ -21,4 +22,7 @@
     "StrategyUnableToProcess",
     "SubstitutionStrategy",
     "compose",
+    "create_composite_source",
+    "create_time_index",
+    "set_priority_coords",
```

Comment:
Do we need `create_time_index` and `set_priority_coords` in the public API? I'd think users only need to know about `create_composite_source`.

Reply:
Maybe there was an issue with imports when importing directly from the files but I'm not sure. I'll try if it works.